policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/sk_senior_supplementary/age_threshold.yaml

```diff
@@ -0,0 +1,9 @@
+description: Saskatchewan provides the senior supplementary tax credit for filers of or above this age threshold.
+values:
+  2023-01-01: 65
+metadata:
+  unit: year
+  label: Saskatchewan senior supplementary tax credit eligible age-threshold
+  reference:
+  - title: Saskatchewan senior supplementary tax credit eligible age-threshold
```

Comment:
change to title of the source: f.e. 2023 Saskatchewan TD1SK Personal Tax Credits Return

Reply:
also add page of resource where numeric value can be found